assets/shaders/postprocess.vert

```diff
@@ -0,0 +1,12 @@
+#version 450
+#extension GL_ARB_separate_shader_objects : enable
+
+layout(location = 0) in vec2 vPositions;
```

Comment:
We could propably infer UVs from the positions

Reply:
Done, actually I move vertex generation in the shader. SO no more index/vertex buffers for the post process quad